hera_qm/tests/test_utils.py

```diff
@@ -369,7 +363,7 @@ def test_apply_yaml_flags_uvcal(filein, new_metadata):
         uvc.antenna_positions = None
         uvc.lst_array = None
 
-    uvc = utils.apply_yaml_flags(uvc, test_flag)
+    uvc = utils.apply_yaml_flags(uvc, test_flag, unflag_first=True)
```

Comment:
Do you actually want to show that previous flags were removed in this test?

Reply:
I've added a check.